### PR TITLE
Erc20 input & output decimals

### PR DESCRIPTION
**Pull request by bergauz**

### CODE REVIEW

File: src/app/modules/wallet/wallet-send/wallet-send.component.html [hunk 1]
```diff
@@ -148,7 +148,15 @@ <h5>Interact with a Smart Contract on GoChain</h5>
                      *ngFor="let functionInput of selectedFunction.inputs; let i=index"
                      class="uk-margin">
                   <div class="uk-form-controls">
-                    <input class="uk-input" placeholder="{{functionInput.name}}" formControlName="{{i}}">
+                    <ng-container *ngIf="selectedFunction.name === 'transfer' && functionInput.name === 'value' && addr.token_symbol; else regularFunctionInput">
+                      <input type="number" class="uk-input" placeholder="Value ({{addr.token_symbol}})" formControlName="{{i}}" #valueInput>
+                      <p *ngIf="valueInput.value">
+                        Raw value: {{valueInput.value * ('1e' + addr.decimals)}}
```
Comment (jmank88) [inline src/app/modules/wallet/wallet-send/wallet-send.component.html]: Do we need to use the big number library here?

**Comment by jmank88** [comment]
Can you add some screenshots?

File: src/app/modules/wallet/wallet-send/wallet-send.component.html (hunk 1 above)
Comment (bergauz) [reply src/app/modules/wallet/wallet-send/wallet-send.component.html]: @jmank88  actually no, cause we miltiply

Comment (jmank88) [reply src/app/modules/wallet/wallet-send/wallet-send.component.html]: But doesn't `1000000000000000000` overflow?

**Comment by bergauz** [comment]
@jmank88 
Screenshot 2019-05-10 at 19 26 29
Screenshot 2019-05-10 at 19 26 15
Screenshot 2019-05-10 at 19 27 02
Screenshot 2019-05-10 at 19 28 20

File: src/app/modules/wallet/wallet-send/wallet-send.component.html (hunk 1 above)
Comment (jmank88) [reply src/app/modules/wallet/wallet-send/wallet-send.component.html]: Or worse:
`10000.000000000001`
`10000.000000000000000001`

Comment (jmank88) [reply src/app/modules/wallet/wallet-send/wallet-send.component.html]: Quick JS playground code shows those inputs losing precision and displaying as exponents, but I'm out of my element here:
`1.0000000000000002e+22`
`1e+22`

Comment (bergauz) [reply src/app/modules/wallet/wallet-send/wallet-send.component.html]: Added bignumber 

Screenshot 2019-05-10 at 22 37 21

Comment (jmank88) [reply src/app/modules/wallet/wallet-send/wallet-send.component.html]: Where was bignumber added? Do we need another separate PR now?
Can you try these numbers as _inputs_:
`10000.000000000001`
`10000.000000000000000001`

Comment (jmank88) [reply src/app/modules/wallet/wallet-send/wallet-send.component.html]: #301

**Pull request merged**